src/persist-client/src/cfg.rs

```diff
@@ -212,6 +214,7 @@ impl PersistConfig {
             compaction_queue_size: 20,
             compaction_yield_after_n_updates: 100_000,
             consensus_connection_pool_max_size: 50,
+            consensus_connection_pool_max_wait: Duration::from_secs(5),
```

Comment:
how stressed are we that this isn't hooked up to LD? on one hand it definitely should finish within 5s, on the other we're gonna feel silly if we have to put out a patch deploy for this. maybe start with something like 60s if you don't feel like connecting this all the way up (and I don't blame you)?

Reply:
this is a pain, but for maximum safety but I wouldn't mind seeing this as a configurable `Option<Duration>` so we can also disable the wait timeout if needed (by setting `None` as we have right now)